http_opa/authorizer.go

```diff
@@ -78,7 +78,7 @@ func (a *httpAuthorizer) Evaluate(ctx context.Context, endpoint string, req inte
 	bearer, err := util.GetBearerFromRequest(req.(*http.Request))
 	if err != nil {
 		logger.WithError(err).Error("get_bearer_from_request")
-		return false, ctx, exception.ErrForbidden
```

Comment:
This could still be forbidden. What are returning in case of grpc middleware?

Reply:
In grpc, the error is suppressed by default.
<img width="635" alt="Screenshot 2024-02-16 at 12 35 37 PM" src="https://github.com/infobloxopen/atlas-authz-middleware/assets/154232025/d978bbdb-c0d3-456d-bf7a-ceff80d3f89a">

Since the header was malformed, I believed it was apt to return `ErrInvalidArgs`